src/app/_components/SiteLayout.tsx

```diff
@@ -1,14 +1,22 @@
+import { Archivo } from 'next/font/google'
+
 import { BreakpointDebugger } from '@/components/_dev_BreakpointDebugger'
 import { Footer } from '@/components/Footer'
 import { Navigation } from '@/components/Navigation'
 import { NetlifyIdentityManager } from '@/components/NetlifyIdentityManager'
 
 import { LayoutProps } from '@/layout'
 
+const archivo = Archivo({
+  weight: ['300', '400', '500', '600', '700'],
+  subsets: ['latin'],
+  display: 'swap',
+})
```

Comment:
Think we don't need to specify weight since it's a variable font

Reply:
Agh! I only saw `approved` and not this message. 🫣 I'll open a new PR to remove it. Thank you.